src/Ganeti/HTools/Node.hs

```diff
@@ -344,7 +349,6 @@ create name_init mem_t_init mem_n_init mem_f_init
        , peers = P.empty
        , rMem = 0
        , rMemForth = 0
-       , pMem = fromIntegral mem_f_init / mem_t_init
```

Comment:
The value is updated in Loader.updateMemStat after the instances are loaded. Without knowing instance sizes we can't determine if state-of-world or state-of-record is smaller, thus we can't know the percentage of guaranteed free memory.